Directory.Build.props

```diff
@@ -124,52 +124,70 @@
 
   <!-- version number information -->
   <PropertyGroup>
-    <IosTargetFrameworkVersion>13.6</IosTargetFrameworkVersion>
-    <TvosTargetFrameworkVersion>13.6</TvosTargetFrameworkVersion>
-    <MacCatalystTargetFrameworkVersion>13.5</MacCatalystTargetFrameworkVersion>
-    <MacosTargetFrameworkVersion>10.14</MacosTargetFrameworkVersion>
-    <AndroidTargetFrameworkVersion>30.0</AndroidTargetFrameworkVersion>
+    <!-- Current .NET -->
+    <IosTargetFrameworkVersion>17.0</IosTargetFrameworkVersion>
+    <TvosTargetFrameworkVersion>17.0</TvosTargetFrameworkVersion>
+    <MacCatalystTargetFrameworkVersion>17.0</MacCatalystTargetFrameworkVersion>
+    <MacosTargetFrameworkVersion>14.0</MacosTargetFrameworkVersion>
+    <AndroidTargetFrameworkVersion>34.0</AndroidTargetFrameworkVersion>
     <WindowsTargetFrameworkVersion>10.0.19041.0</WindowsTargetFrameworkVersion>
     <Windows2TargetFrameworkVersion>10.0.20348.0</Windows2TargetFrameworkVersion>
-    <TizenTargetFrameworkVersion>6.5</TizenTargetFrameworkVersion>
+    <TizenTargetFrameworkVersion>7.0</TizenTargetFrameworkVersion>
+    <!-- Previous .NET -->
+    <IosPreviousTargetFrameworkVersion>16.1</IosPreviousTargetFrameworkVersion>
+    <TvosPreviousTargetFrameworkVersion>16.1</TvosPreviousTargetFrameworkVersion>
+    <MacCatalystPreviousTargetFrameworkVersion>16.1</MacCatalystPreviousTargetFrameworkVersion>
+    <MacosPreviousTargetFrameworkVersion>13.0</MacosPreviousTargetFrameworkVersion>
+    <AndroidPreviousTargetFrameworkVersion>33.0</AndroidPreviousTargetFrameworkVersion>
+    <WindowsPreviousTargetFrameworkVersion>10.0.19041.0</WindowsPreviousTargetFrameworkVersion>
+    <Windows2PreviousTargetFrameworkVersion>10.0.20348.0</Windows2PreviousTargetFrameworkVersion>
+    <TizenPreviousTargetFrameworkVersion>7.0</TizenPreviousTargetFrameworkVersion>
```

Comment:
Are there some projects that use .NET 7? Can we get rid of those, and only target the current supported version? 7 support ended in May.

Reply:
Not sure, but I think that would be a breaking change? The current nugets have .NET 7 so I am not sure if we _can_ remove net7.

Reply:
Seems we still have a lot of people using .net 7 since it still is in VS